src/jit/regalloc.cpp

```diff
@@ -678,7 +678,7 @@ regNumber     Compiler::raUpdateRegStateForArg(RegState *regState, LclVarDsc *ar
 
 #if FEATURE_MULTIREG_STRUCT_ARGS
 #ifdef _TARGET_ARM64_
-    if (argDsc->lvOtherArgReg != REG_NA)
+    if ((argDsc->lvOtherArgReg != REG_STK) && (argDsc->lvOtherArgReg != REG_NA))
```

Comment:
No I believe that Lubo recent made this change because lvArgReg can be REG_STK.
For instance when an incoming argument comes into the method in a register but whose home location is the stack frame, typically because its not used very often in the method.
It also seem possible to me that if we have a 16-byte incoming struct that comes in using two registers that we could possibly be place one or the other part in the stack.